tests/infer/test_enum.py

```diff
@@ -437,14 +437,15 @@ def guide():
     ]))
 
 
-@pytest.mark.parametrize("enumerate1", [None, "sequential", "parallel"])
-@pytest.mark.parametrize("enumerate2", [None, "sequential", "parallel"])
-@pytest.mark.parametrize("enumerate3", [None, "sequential", "parallel"])
+@pytest.mark.parametrize("num_samples", [None, 10000])
```

Comment:
Could we keep the old `enumerate=None` params? This isn't horribly ugly:
```py
ELBO_BERN_PARAMS = [
    (e1, e2, e3, num_samples)
    for e1 in [None, "sequential", "parallel"]
    for e2 in [None, "sequential", "parallel"]
    for e3 in [None, "sequential", "parallel"]
    for num_samples in [None, 10000]
    if num_samples is None or None not in (e1, e2, e3)
]
@pytest.mark.parametrize("enumerate1,enumerate2,enumerate3,num_samples",
                         ELBO_BERN_PARAMS)
...
```

Reply:
You might even want to keep the whole grid for tmc sampling. IIRC this test is cheap.